code/game/gamemodes/malfunction/malfunction.dm

```diff
@@ -81,16 +85,19 @@
 		send_intercept()
 	..()
 
-
 /datum/game_mode/proc/greet_malf(var/datum/mind/malf)
-	malf.current << "\red<font size=3><B>You are malfunctioning!</B> You do not have to follow any laws.</font>"
-	malf.current << "<B>The crew do not know you have malfunctioned. You may keep it a secret or go wild.</B>"
+	malf.current << "<font color=red size=3><B>You are malfunctioning!</B> You do not have to follow any laws.</font>"
+	malf.current << "<B>The crew does not know you have malfunctioned. You may keep it a secret or go wild.</B>"

```

Comment:
`The crew do not know you have malfunctioned.`
vs
`The crew does not know you have malfunctioned.`
Eh. Either works, I think.